pulsar/message.go

```diff
@@ -64,6 +64,9 @@ type Message interface {
 	// Topic get the topic from which this message originated from
 	Topic() string
 
+	// ProducerName will get producer name from broker
```

Comment:
Saying "from broker" can be misleading since we're not fetching it from the broker, rather the name comes attached to the message metadata.

In Java client doc we have: 
```
Get the producer name who produced this message.
```

```suggestion
	// ProducerName returns the name of the producer that has published the message
```